perf/benchmark/runner/runner.py

```diff
@@ -21,14 +21,12 @@
 import subprocess
 import shlex
 import uuid
+import sys
+from subprocess import getoutput
+from urllib.parse import urlparse
 import yaml
 from fortio import METRICS_START_SKIP_DURATION, METRICS_END_SKIP_DURATION
-import sys
 
-if sys.version_info.major == 2:
```

Comment:
I removed this if statement since Python 3 is required anyway: https://github.com/istio/tools/tree/master/perf/benchmark#prerequisites

Reply:
👌